include/libtorrent/piece_picker.hpp

```diff
@@ -450,9 +450,9 @@ namespace libtorrent {
 			, typed_bitfield<piece_index_t> const& have
 			, picker_options_t options) const;
 
-		// only defined when TORRENT_PICKER_LOG is defined, used for debugging
-		// unit tests
+#ifdef TORRENT_PICKER_LOG
```

Comment:
this define is not really a project wide configuration option. It's just defined in piece_picker.cpp when debugging. So I don't think this will make any real difference

Reply:
true, I will revert that